src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -126,6 +130,11 @@ public void reinviteUserScenarios() {
         reinviteSuperUserWithinOneHourShouldReturnTooManyRequest();
     }
 
+    public void updatePbaScenarios() {
+        updatePaymentAccountsShouldReturnSuccess();
+        updatePaymentAccountsShouldReturnPartialSuccess();
```

Comment:
did not see toggle enabled test cases , and also looks like toggle feature is not implemented.

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1096/commits/180cdd1b60baf593e8e2679ac93354ee03d84ee7